advocacy_docs/partner_docs/CommVaultGuide/04-ConfiguringTheIntegration.mdx

```diff
@@ -0,0 +1,108 @@
+---
+title: 'Configuring the Integration'
+description: 'Walkthrough on configuring Commvault Backup & Recovery'
+---
+
+Implementing Commvault Backup & Recovery with EDB Postgres Advanced Server requires the following components:
+
+- EDB Postgres Advanced Server
+- Commvault Backup & Recovery software
+
+## Prerequisites
+
+- A running EDB Postgres Advanced Server instance.
+- Commvault Backup & Recovery installed.
+- EDB Postgres Advanced Server application path and library directory path (e.g. c:\Program Files\edb\as13\bin and c:\Program Files\edb\as13\lib).
+- The login credentials used to access the EDB Postgres Advanced Server database.
+- EDB Postgres Advanced Server archive log directory configured.
+
+## Configure Commvault Backup & Recovery for EDB Postgres Advanced Server
```

Comment:
Heading now
Configure Commvault Backup & Recovery for EDB Postgres Advanced Server or EDB Postgres Extended Server